File: product-webapp/webapp/src/app/Model/personalDetails.ts
import { address } from './address';
import { profilePhoto } from './profilePhoto';
import { vehicleDetails } from './vehicleDetails';

export class personalDetails {
  public firstName: any;
  public lastName: any;
  public dob: any;
  public gender: any;
  public contactNo: any;
  public address = new address();
  public userType: any;
  public vehicleDetails = new vehicleDetails();
  constructor() {}
}

File: user-registration-service/src/main/java/com/niit/registration/model/personalDetails.java
package com.niit.registration.model;

import lombok.*;

import java.util.Set;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class personalDetails {
	private String firstName;
	private String lastName;
	private String DOB;
	private String gender;
	private long contactNo;
	private UserType userType;
	private Address Address;
	private Vehicle vehicleDetails;

}
